pkg/service/test/test.go

```diff
@@ -220,6 +220,10 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 		t.logger.Error("failed to initialise the test", zap.Error(err))
 		return false
 	}
+	// Check if no testset found
+	if len(initialisedValues.Sessions) == 0 {
+		t.logger.Info("no testcases found. Use record mode to record testcases.")
```

Comment:
In the sentence I think rather than `record mode` it is better to use `record command`